ceph/cephfs/cephfs-provisioner.go

```diff
@@ -122,7 +126,13 @@ func (p *cephFSProvisioner) Provision(options controller.VolumeOptions) (*v1.Per
 	user := fmt.Sprintf("kubernetes-dynamic-user-%s", uuid.NewUUID())
 	// provision share
 	// create cmd
-	cmd := exec.Command(provisionCmd, "-n", share, "-u", user)
+	args := []string{"-n", share, "-u", user}
+	if p.setQuota {
+		capacity := options.PVC.Spec.Resources.Requests[v1.ResourceName(v1.ResourceStorage)]
+		requestBytes := strconv.FormatInt(capacity.Value(), 10)
+		args = append(args, "-s", requestBytes)
```

Comment:
also fix the comment here
https://github.com/silenceshell/external-storage/blob/ae906166db5d758bf2de9abefe8488b2951ee7cd/ceph/cephfs/cephfs-provisioner.go#L188